Resources/Private/Layouts/Login.html

```diff
@@ -0,0 +1,163 @@
+										<div class="alert alert-danger">
+											<strong><f:translate key="login.error.message" /></strong>
+											<p><f:translate key="login.error.description" /></p>
+										</div>
+									</div>
+								</f:if>
+								<noscript>
+									<f:be.infobox message="{f:translate(key: 'login.error.javascript')}" state="2" />
+								</noscript>
+								<div class="hidden t3js-login-error-nocookies">
+									<f:be.infobox message="{f:translate(key: 'login.error.cookies')}" state="2" />
+								</div>
+								<div class="typo3-login-form t3js-login-formfields">
+									<form action="?loginProvider={loginProviderIdentifier}" method="post" name="loginform" id="typo3-login-form">
+										<f:form.hidden name="login_status" value="login" />
+										<f:form.hidden name="userident" id="t3-field-userident" class="t3js-login-userident-field" value="" />
+										<f:form.hidden name="redirect_url" value="{redirectUrl}" />
+										<f:form.hidden name="loginRefresh" value="{loginRefresh}" />
+
+										<f:render partial="Login/InterfaceSelector" arguments="{_all}" />
+
+										<f:render section="loginFormFields" />
+
+										<div class="form-group" id="t3-login-submit-section">
+											<mfc:ifTooManyAuthenticationFailures>
+												<f:then>
+													<button class="btn btn-block btn-login t3js-login-submit g-recaptcha"
+															data-callback="onLoginCaptchaSubmit"
+															data-sitekey="{sitekey}"
+															id="t3-login-submit" name="commandLI" data-loading-text="<i class='fa fa-circle-o-notch fa-spin'></i> {f:translate(key: 'login.process')}" autocomplete="off">
+														<f:translate key="login.submit" />
+													</button>
+												</f:then>
+												<f:else>
+													<button class="btn btn-block btn-login t3js-login-submit"
+															id="t3-login-submit" type="submit" name="commandLI" data-loading-text="<i class='fa fa-circle-o-notch fa-spin'></i> {f:translate(key: 'login.process')}" autocomplete="off">
+														<f:translate key="login.submit" />
+													</button>
+												</f:else>
+											</mfc:ifTooManyAuthenticationFailures>
+										</div>
+										<ul class="list-unstyled typo3-login-links">
+											<f:for each="{loginProviders}" as="provider" key="providerKey">
+												<f:if condition="{provider.label}">
+													<f:if condition="{loginProviderIdentifier} != {providerKey}">
+														<li class="t3js-loginprovider-switch" data-providerkey="{providerKey}"><a href="?loginProvider={providerKey}"><i class="fa fa-fw {provider.icon-class}"></i> <span><f:translate key="{provider.label}" /></span></a></li>
+													</f:if>
+												</f:if>
+											</f:for>
+										</ul>
+									</form>
+								</div>
+							</f:then>
+							<f:else>
+								<form action="../../../../../../index.php" method="post" name="loginform" id="typo3-login-form">
```

Comment:
I think an absolute path would be the better why, or?

Reply:
The core does it even worth by only having "index.php"